### ISSUE TITLE
Connector Settings Table too wide

### ISSUE THREAD

**Issue by tealjulia** [airbytehq/airbyte#27357]
Reported by @erohmensing.  Slack thread for context:
https://airbytehq-team.slack.com/archives/C03088BTMFC/p1686697981258989

**Comment by josephkmh** [comment]
Refining notes:
- Column width of some column is probably fixed to a too-large width

**Comment by tealjulia** [comment]
@erohmensing are you able to reproduce this still?  I could not, I think it may have been resolved.

(cc: @dizel852 who probably fixed it)

**Comment by dizel852** [comment]
@tealjulia unfortunately I don't have access to the slack context. 
I guess this is the issue @erohmensing meant, was able to repro on Mac's screen:

https://www.loom.com/share/9144e04819cb4669a237fb40b7d50d24?sid=93d37944-29a1-41b8-8ce5-ccfe06bbdf89

**Comment by tealjulia** [comment]
Ah.  Yeah.  It was that but even worse 😅.  Maybe the connector name is part of the problem and mine just wasn't long enough.

**Comment by erohmensing** [comment]
Yeah, I'd specifically click the workspace I linked in the slack thread! I can still repro it

**Comment by erohmensing** [comment]
(in the one i linked it _barely_ fits if my window is full screened but it is often a little smaller than that, and I think we'd run into it full screen if the name was a little longer)

**Comment by tealjulia** [comment]
I think this is also resulting in the "Create a Connector" button on this page being off screen if the table is too wide.
